src/link/error/index.ts

```diff
@@ -60,9 +60,10 @@ export function onError(errorHandler: ErrorHandler): ApolloLink {
               networkError,
               //Network errors can return GraphQL errors on for example a 403
               graphQLErrors:
-                networkError &&
-                networkError.result &&
-                networkError.result.errors,
+                (networkError &&
+                  networkError.result &&
+                  networkError.result.errors) ||
+                undefined,
```

Comment:
```suggestion
                void 0,
```

We tend to use `void 0` in the codebase so lets use that here 🙂